cfgov/v1/models/feedback.py

```diff
@@ -0,0 +1,86 @@
+from django.db import models
+from django.db.models import Q
+from django.utils.encoding import force_text
+from django.utils.timezone import localdate
+
+from wagtail.wagtailcore.models import Page
+
+from backports import csv
+
+
+class FeedbackQuerySet(models.QuerySet):
+    def for_pages(self, pages, exclude=False):
+        q = self._filter_by_pages_q(pages)
+
+        if exclude:
+            q = ~q
+
+        return self.select_related('page').filter(q)
+
+    def _filter_by_pages_q(self, pages):
+        q = Q()
+
+        for page in pages:
+            q |= self._filter_by_page_q(page)
+
+        return q
+
+    def _filter_by_page_q(self, page):
+        return (
+            Q(page__path__startswith=page.path) &
+            Q(page__depth__gte=page.depth)
+        )
+
+    def write_csv(self, f):
+        headings = [
+            'comment',
+            'currently_own',
+            'expect_to_buy',
+            'email',
```

Comment:
I know these three columns are present in the current Django admin implementation of the feedback downloads, but I *think* they might be old responses that we no longer collect, they're not specified in our current instructions for processing the feedback, and they're pretty much always blank in my experience (except for junk comments where weird formatting forces things to bleed into these columns). I'll check around and see if we're still using them or not.

Reply:
@niqjohnson regarding the presence of these columns, as well as the suggestion from @Scotchester below about adding the URL path: while it would be great to revisit the _content_ of the CSVs, we do have some downstream code (not in this repository) that relies on the current columns being there. So I might prefer to leave that work out of this particular change, especially since the presence of these columns don't affect the complexity much.